to_do.js

```diff
@@ -0,0 +1,22 @@
+const list = document.querySelector('.uList');
+debugger
+// delete item
+// list.addEventListener('click', (event) => {
+//   if(e.target.className == 'delete'){
+//     const li = e.target.parentElement;
+//     li.parentNode.removeChild(li);
+//   }
+// });
+
+const forms = document.querySelector('.form');
+debugger
+Array(...forms).forEach(forms) => {
```

Comment:
No need to Array forms since it is only one element. querySelectorAll is the NodeList that you might have to use Array for.